.gitlab-ci/packet.yml

```diff
@@ -153,6 +153,11 @@ packet_almalinux8-calico:
   extends: .packet_pr
   when: on_success
 
+packet_rockylinux8-calico:
+  stage: deploy-part2
+  extends: .packet_pr
+  when: on_success
```

Comment:
Failed
```
TASK [packet-ci : Template vm files for CI job] ********************************
task path: /builds/kargo-ci/kubernetes-sigs-kubespray/tests/cloud_playbooks/roles/packet-ci/tasks/create-vms.yml:13
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'rockylinux8'
failed: [localhost] (item=1) => {"ansible_index_var": "vm_id", "ansible_loop_var": "item", "changed": false, "item": 1, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'rockylinux8'", "vm_id": 0}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'rockylinux8'
failed: [localhost] (item=2) => {"ansible_index_var": "vm_id", "ansible_loop_var": "item", "changed": false, "item": 2, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'rockylinux8'", "vm_id": 1}
PLAY RECAP *********************************************************************
localhost                  : ok=4    changed=1    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```

Reply:
Yes the failing part is :
```
        - name: cloudinitvolume
          cloudInitNoCloud:
            userDataBase64: {{ cloud_init[cloud_image] }}
```
You need to update ->  https://github.com/kubernetes-sigs/kubespray/blob/master/tests/cloud_playbooks/roles/packet-ci/defaults/main.yml#L21

Reply:
@floryut Thanks for your explanation, I got it.
I will add an entry for rockylinux-8 into the file.